src/functions/products/getProductById/getProductById.ts

```diff
@@ -0,0 +1,24 @@
+import { formatJSONResponse } from "../../../libs/responseModify";
+import { productList } from "../../../mocks/productsList";
+import { middyfy } from "../../../libs/lambda";
+// import { ValidatedEventAPIGatewayProxyEvent } from "../../../types";
+// import schema from "./schema";
+
+export const getProductById = async (event) => {
+  try {
+    const { id } = event.pathParameters;
```

Comment:
we can do const { id } = +event.pathParameters; and forget about + in other places